app/controllers/organizations_controller.rb

```diff
@@ -16,6 +18,45 @@ def new
     @organization = Organization.new
   end
 
+  def new_account
+  end
+
+  def new_category
+  end
+
+  def finish
+    session[:new_organization] = true
+    redirect_to root_path
+  end
+
+  def default_account
+    BankAccount.create(
```

Comment:
@dissident
`config/default_values.yml`
```yaml
bank_accounts:
  - Current
categories:
  - Income
  - Expense
  - Whatever
```
In `config/initializers/default_values.rb`:
```rb
DEFAULT_VALUES = HashWithIndiferentAccess.new(YAML.load(File.join(Rails.root, 'config', 'default_values.yml`)))
```
In `app/model/bank_account.rb`
```
def self.create_defaults(organization)
   [*DEFAULT_VALUES[:bank_accounts]].each{ |ban| o.bank_account.find_or_create_by(name: ban) }
end
```

Something may be wrong but I think you have get idea.